llama-index-core/llama_index/core/agent/react/prompts.py

```diff
@@ -19,10 +18,10 @@
 {tool_desc}
 
 ## Output Format
-To answer the question, please use the following format.
+Please answer in the same language as the question and use the following format:
 
 ```
-Thought: I need to use a tool to help me answer the question.
+Thought: The current language of the user is: (user's language). I need to use a tool to help me answer the question.
```

Comment:
is user's language passed as a parameter?

Reply:
Nope. In our tests, the LLM (we used Mixtral) generated a thought with the language of the user.